multicompany_property_base/models/abstract.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2017 Creu Blanca
+# Copyright 2017 Eficent Business and IT Consulting Services, S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import models, fields, api, _
+from odoo.exceptions import MissingError
+
+
+class ModelProperty(models.AbstractModel):
+    _name = 'model.property'
+
+    company_id = fields.Many2one(
+        comodel_name='res.company',
+        string='Company',
+        required=True,
+        readonly=True
+    )
+
+    @api.multi
+    def _compute_property_fields(self):
+        raise MissingError(_('It must be redefined'))
+
+    # This is the function we will extend in order to generate the information
+    @api.multi
+    def get_property_fields(self, object, properties):
+        raise MissingError(_('It must be redefined'))
```

Comment:
not sure about the error message: not crystal clear. Do you mean "Method not yet defined"?